ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -163,7 +163,8 @@ type RayClusterReconcilerOptions struct {
 
 // Reconcile used to bridge the desired state with the current state
 func (r *RayClusterReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
-	ctx = ctrl.LoggerInto(ctx, r.Log) // TODO: add request namespace here
+	logger := r.Log.WithValues("RayCluster", request.NamespacedName)
```

Comment:
Refer to https://github.com/kubernetes-sigs/controller-runtime/blob/main/TMP-LOGGING.md.